doc/ADR/homebrew_tap.md

```diff
@@ -0,0 +1,71 @@
+# Homebrew tap instructions
+
+## User instructions
+
+1. Add the CBMC tap using the following command:
+
+   ```sh
+   $ brew tap diffblue/cbmc
+   ```
+
+2. Install any of the available CBMC versions in the following manner:
+
+   ```sh
+   $ brew install diffblue/cbmc/cbmc@5.54.0
+   ```
+
+## Developer instructions
```

Comment:
Does "developer" mean cbmc maintainer in this context? What are these instructions supposed to instruct someone to do?